crates/irust_repl/irust_kernel/irust_kernel/resources/__init__.py

```diff
@@ -0,0 +1,4 @@
+import pathlib
+
+
+LOGO_PATH = pathlib.Path(__file__).resolve().parent / "logo-svg.svg"
```

Comment:
Does this have to be in its own file? if it can be in install.py its better

Reply:
I think it can be also in install.py and either will work.